src/main/java/me/karakelley/http/server/RequestReaderFactory.java

```diff
@@ -1,9 +1,12 @@
 package me.karakelley.http.server;
 
+import me.karakelley.http.authorization.Authorizer;
+import me.karakelley.http.RequestValidator;
+
 import java.io.InputStream;
 
 public class RequestReaderFactory {
   public HttpRequestReader getReader(InputStream inputStream) {
-    return new HttpRequestReader(inputStream);
+    return new HttpRequestReader(inputStream, new RequestValidator(new Authorizer("admin", "chicago32")));
```

Comment:
I definitely like that you're injecting the required username and password. That said, I wonder if you could bring that injection up even higher. If I was deploying your server in dev, staging, and production environments and wanted different creds for each environment to keep goofballs out of prod, how might I do that? (Note: don't go overboard with this—I don't need a yaml config file or environment variables... but I think there's probably some compromise between this and that)